pkg/api/core/cleanup/service_cleanup.go

```diff
@@ -0,0 +1,67 @@
+package cleanup
```

Comment:
This service does not follow the service pattern - I just wanted a static service that has no dependencies or interfaces.